ocs_ci/utility/ibmcloud.py

```diff
@@ -337,6 +338,52 @@ class IBMCloud(object):
     Wrapper for Ibm Cloud
     """
 
+    def start_nodes(self, nodes, wait=True):
+        """
+        Start nodes on IBM Cloud.
+
+        Args:
+            nodes (list): The OCS objects of the nodes
+            wait (bool): True for waiting the instances to start, False otherwise
+
+        """
+        if not nodes:
+            raise ValueError("No nodes found to start")
+
+        node_names = [n.name for n in nodes]
+        self.ibmcloud.restart_nodes(nodes)
```

Comment:
```suggestion
        self.restart_nodes(nodes)
```

Did you forgot to remove `ibmcloud` here?